ee/Rules.make

```diff
@@ -44,25 +54,25 @@ EE_SAMPLES := $(EE_SAMPLES:%=$(EE_SAMPLE_DIR)%)
 
 EE_OBJS := $(EE_OBJS:%=$(EE_OBJS_DIR)%)
 
+EE_LINKFILE ?= $(PS2SDKSRC)/ee/startup/src/linkfile
+
+EE_BIN_MAPFILE ?= $(shell basename $(CURDIR)).map
+
+ifneq (x$(EE_COMPACT_EXECUTABLE), x0)
+EE_LDFLAGS += -Wl,-zmax-page-size=128 -s -Wl,--gc-sections
+EE_CFLAGS += -fdata-sections -ffunction-sections
```

Comment:
Do we need `-Os` here as well?

Reply:
I have it set in `EE_OPTFLAGS` in the loader Makefile, but I think it would be better to move it inside Rules.make